### PR TITLE
Added minimal working table test with non-Vector column names

### PR DESCRIPTION
**Pull request by Tokazama**
This is my attempt at a minimal test for non-Vector column names. The two types created `TestVec` and `MinimalTable` aren't fully functional so I wouldn't expect them to do much more than what's occurring in the tests.

### CODE REVIEW

File: test/general.jl [hunk 1]
```diff
@@ -9,32 +9,61 @@
...
+end
+
+Tables.istable(x::MinimalTable) = true
+Tables.columnaccess(::MinimalTable) = true
+Tables.columnnames(x::MinimalTable) = getfield(x, :colnames)
+Tables.columns(x::MinimalTable) = x
+Base.getindex(x::MinimalTable, i1, i2) = getindex(getfield(x, :data), i1, i2)
+Base.getproperty(x::MinimalTable, s::Symbol) = getindex(x, :, findfirst(==(s), Tables.columnnames(x)))
+
+table = MinimalTable([1 2; 3 4], TestVec([:a, :b]))
```
Comment (ronisbr) [inline test/general.jl:33]: Why this is required here?

File: test/general.jl [hunk 2]
```diff
@@ -271,3 +300,8 @@ end
     result   = pretty_table(String, data)
     @test result == expected
 end
+
```
Comment (ronisbr) [inline test/general.jl:303]: Please, remove those empty lines at the end of the file.

File: test/general.jl (hunk 1 above)
Comment (Tokazama) [reply test/general.jl:33]: It's not necessary. I forgot to delete along with the empty lines at the end of the file while writing out some scratch code.

**Comment by ronisbr** [comment]
With the last commit, a lot of test files inside `./test` were created. Please, remove them.

**Comment by Tokazama** [comment]
Goodness, I'm sorry. Trying to do too much at once.

**Comment by ronisbr** [comment]
Well, I have absolutely no idea why the travis test is stale. But everything seems good! Thanks :)

**Pull request merged**